src/chainstate/burn/db/sortdb.rs

```diff
@@ -407,6 +407,32 @@ impl FromRow<DelegateStxOp> for DelegateStxOp {
     }
 }
 
+impl FromRow<PegInOp> for PegInOp {
+    fn from_row<'a>(row: &'a Row) -> Result<Self, db_error> {
+        let txid = Txid::from_column(row, "txid")?;
+        let vtxindex: u32 = row.get_unwrap("vtxindex");
+        let block_height = u64::from_column(row, "block_height")?;
+        let burn_header_hash = BurnchainHeaderHash::from_column(row, "burn_header_hash")?;
+
+        let address = StacksAddress::from_column(row, "address")?;
+        let peg_wallet_address = PoxAddress::from_column(row, "peg_wallet_address")?;
+        let amount = row
```

Comment:
If you're going to panic anyway, can you use `row.get_unwrap()` instead?

Reply:
Sure, I'll use `row.get_unwrap()`. Hmm, looking at the error type it would be even better to return a `Corruption` error. But to be consistent with other functions, I'll write an issue for that as a code quality improvement.

Reply:
Revisiting this, I can't use `row.get_unwrap()` since `amount` is stored as "TEXT" in the database. I prefer to store it as a string rather than an integer, because sqlite does not seem to support 64 bit unsigned integers ([doc](https://www.sqlite.org/datatype3.html#storage_classes_and_datatypes))

Hence, I will do a `get<_, String>.parse().map_err(...)?` sequence here to at least avoid the panic.

Reply:
Addressed in #3486